swaglyrics_backend/issue_maker.py

```diff
@@ -188,10 +187,21 @@ def genius_stripper(song: str, artist: str) -> Optional[str]:
 
 
 @log_args(max_chars=-1)
-def is_title_mismatched(words: List[str], full_title: str, max_err: int) -> bool:
-    mismatch = [word for word in words if word.lower() not in full_title.lower()]
+def is_title_mismatched(title: str, song: str, artist: str, g_song: str, g_artist: str, max_err: int) -> bool:
+    g_title = f"{g_song} by {g_artist}".lower()
+    mismatch = [word for word in title.split() if word.lower() not in g_title.split()]
     logging.debug(f"broke on {mismatch}")
-    return len(mismatch) > max_err
+
+    mismatched = len(mismatch) > max_err
+
+    # handle discord cases where single word song, artist so max_err is 1
+    # TODO: update tests
```

Comment:
If I understand correctly, this todo has been done, right?

https://github.com/SwagLyrics/swaglyrics-backend/pull/36/files#diff-d140bc2bf1860ffeb4862292116fe4b0R266

Reply:
yup, forgot to remove it